docs/src/layouts/links.js

```diff
@@ -98,7 +98,7 @@ const ComponentsLinks = [
         id: 'SNACKBAR',
         route: routes.SNACKBAR,
         children: 'Snackbar',
-        tags: ['android'],
+        tags: ['android', 'native'],
```

Comment:
We should document this for iOS too

Reply:
True, but that's not what this PR is about